nodejs/webapi/swagger/openapiservice.js

```diff
@@ -0,0 +1,63 @@
+const port = 3030;
+
+const swaggerJsdoc = require("swagger-jsdoc");
+const swaggerUi = require("swagger-ui-express");
+
+const description = `[http://localhost:${port}/api-docs/swagger.json](http://localhost:${port}/api-docs/swagger.json)`;
+const serverUrl = `http://localhost:${port}`;
+
+// Basic swagger definition
+const BasicswaggerDefinition = {
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "webapi",
+      description: description,
+      version: "1.0.0",
+    },
+  },
+  apis: ["./server.js"],
+};
+
+// Improved swagger definition
+const ImprovedswaggerDefinition = {
+  definition: {
+    openapi: "3.0.0",
+    info: {
+      title: "Weather Forecast API",
+      description: description,
+      version: "1.0.0",
```

Comment:
Please change it to `v1.0.0` so that it's sync'd with the screenshot for the linting result